### DISCUSSION TITLE
Custom Alias - Fully under User Control & Choice

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#616] [Feature Requests]
When creating Aliases in SimpleLogin using Create Custom Alias or Random by UUID or Words. It creates a Alias like this rechristen_characteristically@simplelogin.fr (Will not work). 

But instead of this how about creating like just rechrist@slmail.com which seems Short & Simple and also easy to remember. What I am Trying to say is the ability to  control the Number of Characters and Remove/Add Special Characters in the Alias but with Simplelogin Domains. 

I hope you get it. Would be Great.

**Comment by nguyenkims** [comment]
As domains like slmail.com are shared between users, the random suffix is added to avoid one person from taking all "nice" aliases like hi@slmail.com, facebook@slmail.com, etc :). If you add your own domain to SimpleLogin, this restriction is removed and you can fully customize the aliases.

**Reply by ghost** [reply]
If So, Why not then make them like No Popular Word or Specific Number of Characters to avoid this ?

The Problem is not with random Suffix, using symbols like _ or . 

Would be great if able to be created without Symbols. Because sometimes symbols in Aliases makes them unable to register in Some Services.

**Reply by nguyenkims** [reply]
Unfortunately at the moment the only way to not have symbols like _, . is to use custom domains.

**Reply by ghost** [reply]
@nguyenkims Atleast may I remove the Symbol if I do have a Premium Account but not a Custom Domain.

**Reply by nguyenkims** [reply]
@Loki-L1130 The limit  is applied for both account types actually. For a full alias customisation, you can also use the new "subdomain" feature.

**Reply by ghost** [reply]
Great, To know about it.
